using/aem-event-setup.md

```diff
@@ -2,7 +2,8 @@
 
 # Setting up AEM Events with Adobe I/O Events
 
-These instructions describe how to set up Adobe Experience Manager (AEM) for Adobe I/O Events. You can use Adobe I/O for notification of AEM events, such as page or asset changes.
+These instructions describe how to set up Adobe Experience Manager (AEM) for Adobe I/O Events. 
+You can use Adobe I/O for notification of AEM events, such as page or asset changes.
```

Comment:
These instructions describe how to set up Adobe Experience Manager (AEM) to integrate with Adobe I/O Events. 
You can use Adobe I/O Events for notification of AEM events, such as page or asset CRUD operations.

Reply:
CRUD operations -> CUD operations